app/service/operation_svc.py

```diff
@@ -42,13 +42,22 @@ def __init__(self):
         except Exception:
             traceback.print_exc()
 
-    """ PRIVATE """
+    async def generate_operation_report(self, op_id, save=False):
+        operations = await self.get_service('data_svc').explode_operation(dict(id=op_id))
```

Comment:
let's make this: operations = (await ...)[0]

Reply:
updated